### PR TITLE
Added AGPL 3

### PR DESCRIPTION
**Pull request by HemantSachdeva**

### CODE REVIEW

File: src/copyrightinserter.ts [hunk 1]
```diff
@@ -21,10 +21,28 @@ export class CopyrightInserter {
...
+License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.`],
+
```
Comment (minherz) [inline src/copyrightinserter.ts]: Please, add the new licenses at the end. There is no need to sort the map initialization data.
The current change creates a cognitive challenge to a reviewer and makes it harder to follow up the change.
Having new map elements added to the end simplifies the diff of the reviewed commit history.

File: src/copyrightinserter.ts [hunk 2]
```diff
@@ -21,10 +21,28 @@ export class CopyrightInserter {
     // map of supported license labels to inline formatting functions that
     // insert copyright holder and year into the license
     // currently supported licenses are:
+    // 'agl3' - AGPL 3.0
```
Comment (minherz) [inline src/copyrightinserter.ts]: nit: you missed 'p' in ag**p**l

**Review by minherz — CHANGES_REQUESTED** [review]
Thank you for taking your time and contributing to this project. There are a couple of changes I'd like to ask you to do in this PR:

1. Address the comment about adding a new value into the `CopyrightMap` map.
1. Please increment the patch version at
https://github.com/minherz/copyright-inserter/blob/f86ac6fc4e7873c6ecb5b3f86c193e2bcc7fd506/package.json#L6
And update the CHANGELOG.md by adding a short description about the change.

**Comment by HemantSachdeva** [comment]
Hope I've resolved all the requested changes :eyes:

**Pull request merged**